cassandra/src/main/scala/akka/stream/alpakka/cassandra/scaladsl/CassandraSink.scala

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.cassandra.scaladsl
+
+import akka.Done
+import akka.stream.scaladsl.{ Flow, Keep, Sink }
+import com.datastax.driver.core.{ BoundStatement, PreparedStatement, Session }
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+import akka.stream.alpakka.cassandra.cassandra._
+
+object CassandraSink {
+  def apply[T](parallelism: Int, statement: PreparedStatement, statementBinder: (T, PreparedStatement) => BoundStatement)(implicit session: Session, ex: ExecutionContext): Sink[T, Future[Done]] =
```

Comment:
some hints for the javadsl
* statementBinder: BiFunction[T, PreparedStatement, BoundStatement]
* delegate to `scaladsl.CassandraSink.apply`
* `mapMaterializedValue` and `scala.compat.java8.FutureConverters._` to convert the `Future` to `CompletionStage`
* Sink.asJava to convert to `akka.stream.javadsl.Sink`